data/list.txt

```diff
@@ -10712,6 +10712,11 @@ gov.scot
 in.net
 web.in
 
+// FAITID : https://faitid.org/
+// Submitted by Maxim Alzoba <tech.contact@faitid.org>
+// https://www.flexireg.net/stat_info
```

Comment:
From where do these comments originate?

Reply:
Those were the comments above the list that originally contained `com.ru`, there are still quite a few domains in the Private section that were under that grouping.

https://github.com/corsearch/ruby-public-suffix/blob/a45afd7fe669bc58f30a378f7dba2279902abae7/data/list.txt#L11693-L11766